controllers/vaultpkisecret_controller.go

```diff
@@ -390,7 +408,7 @@ func (r *VaultPKISecretReconciler) finalizePKI(ctx context.Context, l logr.Logge
 	return nil
 }
 
-func (r *VaultPKISecretReconciler) clearSecretData(ctx context.Context, l logr.Logger, s *secretsv1beta1.VaultPKISecret) error {
+func (r *VaultPKISecretReconciler) clearSecretData(ctx context.Context, _ logr.Logger, s *secretsv1beta1.VaultPKISecret) error {
```

Comment:
Seems like we could remove the logger parameter altogether?

Reply:
Updated in e8d8c00e5a1e2331f9b0bfba5272196e39da397b